plugins/CoreHome/CoreHome.php

```diff
@@ -256,16 +257,11 @@ public function getJsFiles(&$jsFiles)
 
         $jsFiles[] = "plugins/CoreHome/angularjs/content-table/content-table.directive.js";
 
-
         $jsFiles[] = "plugins/CoreHome/angularjs/period-selector/period-selector.directive.js";
         $jsFiles[] = "plugins/CoreHome/angularjs/period-selector/period-selector.controller.js";
 
-        $jsFiles[] = "plugins/CoreHome/angularjs/multipairfield/multipairfield.directive.js";
-        $jsFiles[] = "plugins/CoreHome/angularjs/multipairfield/multipairfield.controller.js";
-
-        $jsFiles[] = "plugins/CoreHome/angularjs/field-array/field-array.directive.js";
-        $jsFiles[] = "plugins/CoreHome/angularjs/field-array/field-array.controller.js";
-
+        // TODO: [Vue] used by RollUpReporting, has to be kept for now.
+        $jsFiles[] = "plugins/CoreHome/angularjs/siteselector/siteselector-model.service.js";
```

Comment:
I thought about using an adapter, but since the data in the angularjs service is meant to be mutable from outside of the service, and the Vue counterpoint is not, I thought it would be better to simply leave it for now.